api/lib/infrastructure/DomainTransaction.js

```diff
@@ -7,7 +7,9 @@ class DomainTransaction {
   async commit() {
     await this.knexTransaction.commit();
   }
-  async rollback() {}
+  async rollback() {
+    await this.knexTransaction.rollback();
+  }
```

Comment:
could be in 9f44c2fcdb38216f6f7caf1575ae252722c6fb54

Reply:
Oui mais à ce stade je n'ai plus le courage de ranger ces commits, d'autant que cette fonction disparaît ensuite.